src/model/Weather.ts

```diff
@@ -0,0 +1,19 @@
+export interface WeatherLocation { 
```

Comment:
This makes a lot of sense! May need to add more data types depending on what we need in future iterations of app.